src/screens/SendCoinsConfirmScreen.tsx

```diff
@@ -119,6 +119,7 @@ class SendCoinsConfirmScreen extends Component<Props> {
                 title: i18n.message.returnToDashboard,
                 onPress: () => {
                   this.navgitateToMainCard();
+                  // @ts-ignore
```

Comment:
why @ts-ignore here?

Reply:
because typescript started requiring an argument in resolve

Reply:
new Promise<void>((resolve, reject) => {